### PR TITLE
#105 chore: reworked keycloak client and added members to get_groups

### PR DESCRIPTION
**Pull request by zsinnema**

### CODE REVIEW

**Comment by zsinnema** [comment]
closes #105

File: libraries/cloudharness-common/cloudharness/auth/keycloak/__init__.py [hunk 1]
```diff
@@ -17,63 +17,20 @@
...
     PORT = getattr(accounts_app,'port')
     USER = getattr(accounts_app.admin,'user')
     PASSWD = getattr(accounts_app.admin,'pass')
 except:
-    AUTH_REALM = 'mnp'
-    AUTH_DOMAIN = 'localhost'
-    HOST = 'localhost'
-    PORT = '8080'
-    USER = 'mnp'
+    AUTH_REALM = 'ifn'
```
Comment (filippomc) [inline libraries/cloudharness-common/cloudharness/auth/keycloak/__init__.py:26]: Need to remove external project mentions

**Review by filippomc — CHANGES_REQUESTED** [review]
Looks good in general, only need to remove project-specific code

**Review by filippomc — APPROVED** [review]
lgtm

**Pull request merged**